Intersect.Editor/Forms/Editors/Events/Event Commands/EventCommand_ConditionalBranch.cs

```diff
@@ -851,6 +868,11 @@ private void lblStringTextVariables_Click(object sender, EventArgs e)
             );
         }
 
+        private void NudItemAmount_ValueChanged(object sender, System.EventArgs e)
+        {
+            if (nudItemAmount.Value < 1) nudItemAmount.Value = 1;
```

Comment:
`nudItemAmount.Value = Math.Max(1, nudItemAmount.Value);`

Reply:
Yes sir, I won't do it again sir. (maybe? Force of habit)

I copied this assuming I changed them before. Haha